.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": ["next", "prettier"],
```

Comment:
We should add `eslint:recommended` as well here (plus a few more plugins). "next", "prettier" are specific rulesets for frameworks.

Happy for this to be done in a follow up PR though

Reply:
I'll do that here in one go

Reply:
Or maybe not, `eslint:recommended` is much more strict, will do that in the following PR